package/Editor/SceneTools/SceneContextMenuActions.cs

```diff
@@ -0,0 +1,270 @@
+        /// </summary>
+        private static void RemoveBackground()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            SpriteRenderer spriteRenderer = selectedLayer.GetComponent<SpriteRenderer>();
+            if (spriteRenderer == null)
+            {
+                Debug.LogError("RemoveBackground: No SpriteRenderer found on the selected GameObject.");
+                return;
+            }
+
+            if (spriteRenderer.sprite == null)
+            {
+                Debug.LogError("RemoveBackground: No sprite found on the SpriteRenderer.");
+                return;
+            }
+
+            Texture2D texture2D = spriteRenderer.sprite.texture;
+            if (texture2D == null)
+            {
+                Debug.LogError("RemoveBackground: No texture found on the sprite.");
+                return;
+            }
+
+            string texturePath = AssetDatabase.GetAssetPath(texture2D);
+            TextureImporter importer = AssetImporter.GetAtPath(texturePath) as TextureImporter;
+
+            if (importer != null)
+            {
+                importer.isReadable = true;
+                importer.textureCompression = TextureImporterCompression.Uncompressed;
+                importer.SaveAndReimport();
+
+                texture2D = AssetDatabase.LoadAssetAtPath<Texture2D>(texturePath);
+                if (texture2D == null)
+                {
+                    Debug.LogError("RemoveBackground: Could not reload the texture after reimporting.");
+                    return;
+                }
+            }
+            else
+            {
+                // Handle runtime-generated textures
+                if (!texture2D.isReadable)
+                {
+                    Debug.LogError("RemoveBackground: The texture is not readable and not an asset texture.");
+                    return;
+                }
+            }
+
+            Texture2D uncompressedTexture = new Texture2D(texture2D.width, texture2D.height, TextureFormat.RGBA32, false);
+            uncompressedTexture.SetPixels(texture2D.GetPixels());
+            uncompressedTexture.Apply();
+
+            string dataUrl = CommonUtils.Texture2DToDataURL(uncompressedTexture);
+            string name = CommonUtils.GetRandomImageFileName();
+            string param = $"{{\"image\":\"{dataUrl}\",\"name\":\"{name}\",\"format\":\"png\",\"returnImage\":\"false\"}}";
+            Debug.Log(param);
+
+            ApiClient.RestPut("images/erase-background", param, response =>
+            {
+                dynamic jsonResponse = JsonConvert.DeserializeObject(response.Content);
+                string imageUrl = jsonResponse.asset.url;
+                CommonUtils.FetchTextureFromURL(imageUrl, texture =>
+                {
+                    // Update the sprite directly in Unity
+                    CommonUtils.ReplaceSprite(spriteRenderer, texture);
```

Comment:
![image](https://github.com/user-attachments/assets/4f922ec9-15bd-49df-97a1-28fc43103357)
Seems something is not pushed on CommonUtils file ??

Reply:
Issue should be solved @TheLLspectre